source/rainerscript/global.rst

```diff
@@ -372,3 +372,25 @@ The following parameters can be set:
   that you found via a web search.
 
   Once again: **users must NOT set this parameter!**
+
+- **oversizemsg.errorfile** [file name] available 8.35.0+
+
+  This parameter is used to specify the name of the oversize message log file.
+  Here messages that are longer than maxMessageSize will be gathered.
+
+- **oversizemsg.input.mode** [mode] available 8.35.0+
+
+  With this parameter the behavior for oversized messages can be specified.
+  Available modes are:
+
+  - truncate: Oversized messages will be truncated.
+  - split: Oversized messages will be split and the rest of the message will
+    be send in another message.
```

Comment:
@PascalWithopf @rgerhards 

This is the typo I was referencing. VERY minor, but wanted to add missing detail from my remark on the PR.